ui/helpers/http.js

```diff
@@ -43,7 +43,11 @@ export const getData = async function getData(url) {
     failureStatus = response.status;
   }
 
-  if (response.headers.get('content-type') === 'text/html' && failureStatus) {
+  const contentType = response.headers
+    .get('content-type')
+    .startsWith('text/html');
+
+  if (contentType && failureStatus) {
```

Comment:
Ah I was thinking the `.startsWith()` would be on `contentType.startsWith()` - as is the `contentType` is really `isTextContentType`?

Anyway not overly important, so happy to leave for now :-)